packages/server/src/client/internal/open-feature-client.ts

```diff
@@ -380,4 +386,23 @@ export class OpenFeatureClient implements Client {
   private get _logger() {
     return this._clientLogger || this.globalLogger();
   }
+
+  private mergeContexts(invocationContext: EvaluationContext) {
+    // merge global and client contexts
+    return {
+      ...OpenFeature.getContext(),
+      ...OpenFeature.getTransactionContext(),
```

Comment:
Maybe instead if using the OpenFeature global API to get the context we could add a context accessor function as we do here: https://github.com/open-feature/js-sdk/pull/1020/files#diff-8a151eb52e8ba25881ad3b837c98a32807dabf7b7f771518248edb6206e5add9R51
I would like this as we would not have a circular dependency between the global API and the client then.

But this would also apply to `OpenFeature.getHooks` used in this class and it might be out of context for this API. So we can resolve this, just wanted to mention it and see if we want to do it now.

Reply:
Great just a nice thing to "clean up"! :)